egs/uw3/v1/local/prepare_lexicon.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+parser = argparse.ArgumentParser(description="""Creates the list of characters and words in lexicon""")
+parser.add_argument('database_path', type=str, help='path to train text file')
+parser.add_argument('test_text', type=str, help='path to test text file to include it in lexicon')
+parser.add_argument('dir', type=str, help='output path')
+args = parser.parse_args()
+
+### main ###
+char = {}
+lex = {}
+
+text_path = os.path.join(args.database_path,'text')
+with open(text_path) as f:
+  for line in f:
+    line = line.strip()
+    line_vect = line.split(' ')
+    for i in range(1,len(line_vect)):
+      characters = list(line_vect[i])
+      entry = " ".join(characters)
+
```

Comment:
Please remove unnecessary empty lines in this script and other scripts.
Only leave the empty lines that you think actually increase readability.